src/api/guardian_filter.js

```diff
@@ -1 +1,13 @@
-module.exports = response => {};
+module.exports = response => {
+  let results = response.results.reduce((articles, article) => {
```

Comment:
any reason why this is let and not a const ?

Reply:
also you could use deconstruction for `(articles, article)` to be `(articles, { fields, id })`   so if you scroll down a bit to my next comment :arrow_down:

Reply:
done